README.md

```diff
@@ -32,7 +32,18 @@ To build for the EV3:
 
 * On your host computer (not in `brickstrap shell`), use NFS or sshfs to share
 <some-path-like-home/user/build-brickman> with your EV3.
-* On your EV3, connect the share and run `./brickman`
```

Comment:
> Should we have some devscript repo?

That is the idea behind <https://github.com/ev3dev/ev3dev-tools>. It needs to get cleaned up and packaged.